test/Tools/NodeDocumentationMarkdownGeneratorTests/MarkdownGeneratorCommandTests.cs

```diff
@@ -164,7 +164,7 @@ public void ProducesCorrectOutputFromCoreDirectory_preloadedbinaries()
             FromDirectoryCommand.HandleDocumentationFromDirectory(opts);
 
             var generatedFileNames = tempDirectory.GetFiles().Select(x => x.Name);
-            Assert.AreEqual(701, generatedFileNames.Count());
+            Assert.AreEqual(707, generatedFileNames.Count());
```

Comment:
@QilongTang do you know why the number has only increased by 6 when in fact there are 21 new nodes?

Reply:
Probably could be related to that we are using just some dlls as nodes source  and also probably some nodes have already documentation, I will continue checking the details.
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/ac6cea7f-e228-43db-b1af-51c91be1074c)